Helpers/EmailSender.cs

```diff
@@ -29,11 +29,12 @@ public void sendEmail(string email, string body)
             mail.Body = body;
             try
             {
-                smtpClient.Send(message: mail);
+                await Task.Run(() => smtpClient.Send(message: mail));
+                return true;
             }
             catch (Exception)
             {
-
+                return false;
```

Comment:
Maybe this exception should be handled better. Like trying to send the email again